_config.yml

```diff
@@ -6,7 +6,9 @@ canonical_domain: www.login.gov
 # i18n
 languages:
   - en
-  # - es # disabled in production for now
+  - es
+  - fr
```

Comment:
Yeah sorry I updated my initial note to reflect that - no French, however Spanish is currently live via the Federalist config.